core/src/main/resources/benchmark-jcache.xml

```diff
@@ -1,5 +1,5 @@
-<!-- RadarGun 2.0 benchmark -->
-<benchmark xmlns="urn:radargun:benchmark:2.0">
+<!-- RadarGun 2.1 benchmark -->
+<benchmark xmlns="urn:radargun:benchmark:2.1">
 
```

Comment:
So this schema reference works based on the schema being embedded in the main plugin schema?